whylog/tests/utils.py

```diff
@@ -1,10 +1,17 @@
 import os.path
+import platform
 
 
 class ConfigPathFactory(object):
     @classmethod
-    def get_path_to_config_files(cls, prefix_path):
+    def get_path_to_config_files(cls, prefix_path, multi_platform=True):
         parsers_path = os.path.join(prefix_path, 'parsers.yaml')
         rules_path = os.path.join(prefix_path, 'rules.yaml')
-        log_type_path = os.path.join(prefix_path, 'log_types.yaml')
+        if multi_platform:
+            if platform.system() == 'Windows':
+                log_type_path = os.path.join(prefix_path, 'windows_log_types.yaml')
```

Comment:
Extract the string to a variable and then DRY on the rest of the line. Also please add a comment which will say what is going on here and why.